### PR TITLE
Update benchmarks

### PR DESCRIPTION
**Pull request by sigkell**
- Add benchmark for Koa + koa-route
- Update the benchmark results in `README.md` to reflect newer framework versions

### CODE REVIEW

**Comment by delvedor** [comment]
Hi! Thank you for contributing!

I’m wondering if it makes sense create a new docs page just for the benchmarks and features comparison with the others framework.

I’m saying this because I saw that you added a benchmark with koa+router, but for instance, fastify offers logging of the box as well.

@mcollina probably we should rewrite the benchmarks and add to all frameworks both routing (if they don’t provide it) and logging. What do you think?

**Comment by sigkell** [comment]
On second thought, would it be better to change `koa-route` for `koa-router`? Latter appears more widely used.

**Comment by mcollina** [comment]
Oh I thought it was a spelling error, it should be koa-router
Il giorno mar 25 lug 2017 alle 04:20 Fionn Kelleher <
notifications@github.com> ha scritto:

File: benchmarks/koa-route.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+var Koa = require('koa')
+var _ = require('koa-route')
```
Comment (sigkell) [inline benchmarks/koa-route.js]: See https://github.com/koajs/route#koa-route

**Comment by delvedor** [comment]
> On second thought, would it be better to change `koa-route` for `koa-router`? Latter appears more widely used.

I'll go for `koa-router` since is definitely more used :)

**Comment by mcollina** [comment]
Are we ok to land this?

**Pull request merged**